eksconfig/add-on-managed-node-groups.go

```diff
@@ -141,6 +141,29 @@ type MNG struct {
 
 	// VersionUpgrade configures MNG version upgarde.
 	VersionUpgrade *MNGVersionUpgrade `json:"version-upgrade,omitempty"`
+
+	// ConfigUpdate configures MNG config update.
+	ConfigUpdates []*MNGConfigUpdate `json:"config-update,omitempty"`
```

Comment:
```
ScaleUpdates []*MNGScaleUpdates `json:"scale-update,omitempty"`
```

Reply:
And can we put `ScaleUpdates` above `VersionUpgrade`? We want to run `VersionUpgrade` always at the end.

Reply:
That way, we can run something like this:

```diff
diff --git a/eks/mng/mng.go b/eks/mng/mng.go
index fd6614e9..3864bdf4 100644
--- a/eks/mng/mng.go
+++ b/eks/mng/mng.go
@@ -124,6 +124,14 @@ func (ts *tester) Create() (err error) {
                }
        }
 
+       for mngName, v := range ts.cfg.EKSConfig.AddOnManagedNodeGroups.MNGs {
+               for _, cur := range v.ScaleUpdates {
+                       if err = ts.scaleTester.Scale(mngName, cur); err != nil {
+                               return err
+                       }
+               }
+       }
+
        ts.cfg.EKSConfig.AddOnManagedNodeGroups.Created = true
        return ts.cfg.EKSConfig.Sync()
 }
```